framework/console/Controller.php

```diff
@@ -488,7 +488,7 @@ protected function parseDocCommentSummary($reflection)
     {
         $docLines = preg_split('~\R~u', $reflection->getDocComment());
         if (isset($docLines[1])) {
-            return trim($docLines[1], ' *');
+            return trim($docLines[1], "\x09 *");
```

Comment:
https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md#7-update-the-changelog